bitnami/mongodb/templates/_helpers.tpl

```diff
@@ -224,6 +224,39 @@ Return true if a configmap object should be created for MongoDB(R) Arbiter
 {{- end -}}
 {{- end -}}
 
+
+
+{{/*
+Return true if the Hidden should be deployed
+*/}}
+{{- define "mongodb.hidden.enabled" -}}
+{{- if and (eq .Values.architecture "replicaset") .Values.hidden.enabled }}
+    {{- true -}}
+{{- else -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the configmap with the MongoDB(R) configuration for the Hidden
+*/}}
+{{- define "mongodb.hidden.configmapName" -}}
+{{- if .Values.hidden.existingConfigmap -}}
+    {{- printf "%s" (tpl .Values.hidden.existingConfigmap $) -}}
+{{- else -}}
+    {{- printf "%s-hidden" (include "mongodb.fullname" .) -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return true if a configmap object should be created for MongoDB(R) Hidden
+*/}}
+{{- define "mongodb.hidden.createConfigmap" -}}
+{{- if and (eq .Values.architecture "replicaset") .Values.hidden.enabled .Values.hidden.configuration (not .Values.hidden.existingConfigmap) }}
+    {{- true -}}
+{{- else -}}
```

Comment:
Could you please remove empty {- else -} statements from lines 235 and 256?

It would be fine just doing:
```
{{- define "my_function" -}}
{{- if condition -}}
    do something
{{- end -}}
{{- end -}}
```